src/tools/binaryen-shell.cpp

```diff
@@ -77,8 +77,8 @@ static void run_asserts(size_t* i, bool* checked, Module* wasm,
   std::unique_ptr<ShellExternalInterface> interface;
   std::unique_ptr<ModuleInstance> instance;
   if (wasm) {
-    interface = make_unique<ShellExternalInterface>();
-    instance = make_unique<ModuleInstance>(*wasm, interface.get());
+    interface = wasm::make_unique<ShellExternalInterface>();
```

Comment:
we do `using namespace wasm`, why are these needed? ambiguity somehow I guess? is there some other make_unique in the global namespace?